docs/docsite/rst/scenario_guides/guide_scaleway.rst

```diff
@@ -0,0 +1,231 @@
+
+How to create a compute instance?
+=================================
+
+Now that we have an SSH key configured, the next step is to spin up a server!
+`scaleway_compute <https://docs.ansible.com/ansible/devel/modules/scaleway_compute_module.html#scaleway-compute>`_  is a module that can create, update and delete Scaleway compute instances:
+
+.. code-block:: yaml
+
+    - name: Create a server
+      scaleway_compute:
+        name: foobar
+        state: present
+        image: ec8b431e-ad39-4523-8b94-f3fa7f3cbd06
+        organization: 951df375-e094-4d26-97c1-ba548eeb9c42
+        region: ams1
+        commercial_type: START1-S
+
+Here are the parameter details for the example shown above:
+
+- ``name`` is the name of the instance (the one that will show up in your web console).
+- ``image`` is the UUID of the system image you would like to use.
+  A list of all images is available for each availability zone.
+- ``organization`` represents the organization that your account is attached to.
+- ``region`` represents the Availability Zone which your instance is in (for this example, par1 and ams1).
+- ``commercial_type`` represents the name of the commercial offers.
+  You can check out the Scaleway pricing page to find which instance is right for you.
+
+Take a look at this short playbook to see a working example using ``scaleway_compute``:
+
+.. code-block:: yaml
+
+    # SCW_TOKEN='XXX' ansible-playbook ./test/legacy/scaleway_compute.yml
+
+    - name: Test compute instance lifecycle on a Scaleway account
+      hosts: localhost
+      gather_facts: no
+      environment:
+        SCW_API_KEY: ""
+
+      tasks:
+
+        - name: Create a server
+          register: server_creation_task
+          scaleway_compute:
+            name: foobar
+            state: present
+            image: ec8b431e-ad39-4523-8b94-f3fa7f3cbd06
+            organization: 951df375-e094-4d26-97c1-ba548eeb9c42
+            region: ams1
+            commercial_type: START1-S
+            wait: true
+
+        - debug: var=server_creation_task
+
+        - assert:
+            that:
+              - server_creation_task is success
+              - server_creation_task is changed
+
+        - name: Run it
+          scaleway_compute:
+            name: foobar
+            state: running
+            image: ec8b431e-ad39-4523-8b94-f3fa7f3cbd06
+            organization: 951df375-e094-4d26-97c1-ba548eeb9c42
+            region: ams1
+            commercial_type: START1-S
+            wait: true
+            tags:
+              - web_server
+          register: server_run_task
+
+        - debug: var=server_run_task
+
+        - assert:
+            that:
+              - server_run_task is success
+              - server_run_task is changed
+
+
+Dynamic Inventory Script
+========================
+
+Ansible ships with `Scaleway Dynamic Inventory Plugin <https://docs.ansible.com/ansible/devel/plugins/inventory/scaleway.html>_`.
```

Comment:
@acozine What should I use here as a ref link for dynamic inventory? It is not a module. Does it have a special semantic to point to plugins?